app/services/resources/search.rb

```diff
@@ -22,14 +22,15 @@ class Search
       end
     end.flatten.join('||').freeze
 
-    def self.perform(query, scope: Resource)
+    def self.perform(query, scope: Resource, sort: sort_order)
       scope
         .joins('LEFT JOIN services ON services.resource_id = resources.id')
         .joins('LEFT JOIN notes ON notes.resource_id = resources.id')
         .joins('LEFT JOIN categories_resources ON categories_resources.resource_id = resources.id')
         .joins('LEFT JOIN categories ON categories.id = categories_resources.category_id')
+        .joins('LEFT JOIN addresses ON addresses.resource_id = resources.id')
         .where("#{CLAUSE} @@ plainto_tsquery('#{SEARCH_CONFIG}', ?)", query)
-        .distinct
+        .order(sort)
```

Comment:
Need to keep the distinctness here but having both on the same query results in 
ActiveRecord::StatementInvalid (PG::InvalidColumnReference: ERROR:  for SELECT DISTINCT, ORDER BY expressions must appear in select list

Need to do this as a subquery I believe?